package.json

```diff
@@ -4,9 +4,12 @@
   "description": "",
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
+  "module": "dist/esm/index.js",
   "scripts": {
     "add-contributor": "kcd-scripts contributors add",
-    "build": "tsc -p tsconfig.build.json",
+    "build": "npm-run-all build:cjs build:esm",
```

Comment:
The reason this is failing it's checks is because the `typecheck:build` script is failing. It passes --noEmit to the `build` script and `npm-run-all` doesn't know what to do with it.

To fix this you can replace the `typecheck:build` script with:
`"typecheck:build": "npm run build:cjs -- --noEmit && npm run build:esm -- --noEmit"`